src/config/env.ts

```diff
@@ -99,6 +100,7 @@ export interface EnvVars {
 	GITHUB_HOSTNAME: string;
 	DEPLOYMENT_DATE: string;
 	BULL_QUEUE_PREFIX?: string;
+	HASH_SECRET: string;
```

Comment:
This should also be part of the required env vars function above to prevent hashing with 'undefined'

Reply:
don't forget this one